### PR TITLE
Update items global with various items/gear sets

### PR DESCRIPTION
**Pull request by CatsEyeXI**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
Pushing a list of items we've added to our `items.lua` over time.

## Steps to test these changes
No concrete test plan. Items should now be able to be referenced as `xi.items.<item>`

### CODE REVIEW

File: scripts/globals/items.lua [hunk 1]
```diff
@@ -759,12 +771,19 @@ xi.items =
     IMPERIAL_MYTHRIL_PIECE          = 2186,
     IMPERIAL_GOLD_PIECE             = 2187,
     FIENDISH_SKIN                   = 2189,
+    APPRAISAL_SWORD                 = 2190,
     UNAPPRAISED_SWORD               = 2190,
+    APPRAISAL_DAGGER                = 2191,
     UNAPPRAISED_DAGGER              = 2191,
+    APPRAISAL_POLEARM               = 2192,
```
Comment (TeoTwawki) [inline scripts/globals/items.lua]: duuuuuuuuuuuuuuuuplicates

File: scripts/globals/items.lua [hunk 2]
```diff
@@ -2701,6 +2744,7 @@ xi.items =
     SANCTUARY_OBI                   = 11765,
     TWILIGHT_HELM                   = 11798,
     FAZHELUO_HELM                   = 11808,
+    FAZHELUO_HELM                   = 11808,
```
Comment (TeoTwawki) [inline scripts/globals/items.lua]: more duplicate

Comment (CatsEyeXI) [reply scripts/globals/items.lua]: duplicates removed >.> attempting once moar

**Comment by TeoTwawki** [comment]
I  assume you have soon to be pr's were you need to use these?

File: scripts/globals/items.lua [hunk 3]
```diff
@@ -4396,6 +4457,10 @@ xi.items =
     FAUCHEUSE                       = 18952,
     GLEAMING_ZAGHNAL                = 18963,
     DIRE_SCYTHE                     = 18964,
+    TERPSICHORE                     = 18989,
```
Comment (Xaver-DaRed) [inline scripts/globals/items.lua]: There are 11 weapons called this. I think we have a special naming convention for this

**Comment by CatsEyeXI** [comment]
> I assume you have soon to be pr's were you need to use these?

Yessir :)

File: scripts/globals/items.lua (hunk 1 above)
Comment (claywar) [reply scripts/globals/items.lua]: These APPRAISAL_x items still need to be removed.  UNAPPRAISED_x is the format we're using here, and already exist

File: scripts/globals/items.lua (hunk 3 above)
Comment (Xaver-DaRed) [reply scripts/globals/items.lua]: Can you rename it to `TERPSICHORE_75` ?

We will have to rename other relics and mythics to align with this, but that is out of the scope of this PR

File: scripts/globals/items.lua (hunk 1 above)
Comment (CatsEyeXI) [reply scripts/globals/items.lua]: > These APPRAISAL_x items still need to be removed. UNAPPRAISED_x is the format we're using here, and already exist

understood, these have been removed.

File: scripts/globals/items.lua (hunk 3 above)
Comment (CatsEyeXI) [reply scripts/globals/items.lua]: > Can you rename it to `TERPSICHORE_75` ?
> 
> We will have to rename other relics and mythics to align with this, but that is out of the scope of this PR

updated to reflect the level

**Review by Xaver-DaRed — APPROVED** [review]
LGTM

**Review by claywar — APPROVED** [review]

**Pull request merged**